bot/cogs/grades_data/grades_cog.py

```diff
@@ -151,7 +151,8 @@ def initialize(self):
             
         else:
             log.info('FILES NOT FOUND')
-
+            raise FileNotFoundError
```

Comment:
We probably want to say WHAT file isnt found here `FileNotFoundError('some file is missing')`